network/server/tchannelthrift/convert/convert_test.go

```diff
@@ -75,8 +75,7 @@ func regexpQueryTestCase(t *testing.T) (idx.Query, *rpc.IdxQuery) {
 func conjunctionQueryATestCase(t *testing.T) (idx.Query, *rpc.IdxQuery) {
 	q1, rq1 := termQueryTestCase(t)
 	q2, rq2 := regexpQueryTestCase(t)
-	q, err := idx.NewConjunctionQuery(q1, q2)
-	require.NoError(t, err)
+	q := idx.NewConjunctionQuery(q1, q2)
```

Comment:
could you add a test case for negation queries in `TestConvertFetchTaggedRequest`